src/EventStore.Core/Services/Archive/ArchiveOptions.cs

```diff
@@ -4,7 +4,6 @@
 namespace EventStore.Core.Services.Archive;
 
 public class ArchiveOptions {
-	public bool Enabled { get; init; } = true;
```

Comment:
i think that it's still useful to have an explicit `Enabled` property for a few reasons:
- it's consistent with the configuration options of other plugins
- a user may want to temporarily disable archiving. to do so, the user can just change `Enabled` to `false` without needing to comment out the whole archive config section (but `StorageType: None` is less easy to remember)

Reply:
yeah true, although the part i was uncomfortable with was having the Enabled bool default to true while having the whole archiving mechanism disabled 'by default' if there is no config. It does make sense but it's a little bit complicated. Would it be alright with you if we set the Enabled bool to false by default? The user would have to explicitly enable it as well as setting the storage type, but it's workable for me